backend/infrahub/computed_attribute/tasks.py

```diff
@@ -169,9 +170,11 @@ async def process_jinja2(
     computed_attribute_kind: str,
     updated_fields: list[str] | None = None,
 ) -> None:
-    """Request to the creation of git branches in available repositories."""
     log = get_run_logger()
     service = services.service
+
+    await add_tags(branches=[branch_name], nodes=[object_id])
```

Comment:
The other changes looks good. I'm just wondering about the expectations with nodes in this case `object_id` could in this case refer to the node where the computed attribute is defined, but it can only be on the related object that is being mutated. In the second scenario we find the actual object after querying a bit further down.

The code would of course be correct in the light of us processing computed attributes on behalf of this specific object_id. Just wanted to check if that was the expectation here?

Reply:
Good point, I've removed the object id for now
Looking more closely at the code the right solution is to create a sub flow for each node that needs to be updated, I'll take care of it in another PR